docs/ROADMAP.md

```diff
@@ -11,8 +11,8 @@ There is an urgent, global need for an official WHO App to help contain and miti
 Version numbering terminology:
 
 * **v0.1**: refers to the initial buildout that will be submitted to the app stores but not intended for public release, aiming for submission on Thursday, March 26, 2020 ([v0.1 milestone](https://github.com/WorldHealthOrganization/app/milestone/2))
-* **v0.2+**: any versions starting with major version v0 are not intended for public distribution or consumption
-* **v1.0**: the first version that we feel offers enough value to be published onto the app stores for public download will be v1.0
+* **v0.2**: an informal preview build for WHO by end of day Friday, March 27, 2020
```

Comment:
does it help to indicate which timezone?

Reply:
Yes absolutely